corehq/apps/userreports/tasks.py

```diff
@@ -231,10 +231,28 @@ def delete_data_source_task(domain, config_id):
 
 
 @periodic_task(
-    run_every=settings.ASYNC_INDICATOR_QUEUE_CRONTAB, queue=settings.CELERY_PERIODIC_QUEUE
+    run_every='*/5', queue=settings.CELERY_PERIODIC_QUEUE
 )
 def run_queue_async_indicators_task():
-    queue_async_indicators.delay()
+    """ASYNC_INDICATOR_QUEUE_TIMES will be of the format:
+    {
+        '*': [(begin_hour, end_hour), (begin_hour, end_hour), ...] catch all for days
+        '1': [(begin_hour, end_hour), ...] hours for Monday (Monday 1, Sunday 7)
+    }
+    All times UTC
+    """
```

Comment:
this is just a style thing, but it might be nice if this code read:

```python
if current_time_in_range(settings.ASYNC_INDICATOR_QUEUE_TIMES):
    queue_async_indicators.delay()
```

a side effect/benefit of that would be that we could write a few tests for `current_time_in_range` and make sure it works as expected.

Reply:
Yeah that would be better. I'll do that in a follow up